forch/port_state_manager.py

```diff
@@ -114,10 +114,10 @@ def __init__(self, process_device_placement, process_device_behavior, get_vlan_f
         self._static_device_behaviors = {}
         self._dynamic_device_behaviors = {}
         self._process_device_placement = (
-            lambda *args, **kwargs: process_device_placement(*args, **kwargs)
+            lambda mac, placement, static: process_device_placement(mac, placement, static)
```

Comment:
why do you need the lambda function at all?  Isn't this equivalent to just saying:

`self._process_device_placement = process_device_placement`

??  Maybe I'm missing what this is doing.

Reply:
So that's mainly because the functions passed in (e.g., process_device_placement) may be None. So using a lambda and if/else here is equivalent to:
```
def _process_device_placement(mac, placement, static):
    if process_device_placement:
        process_device_placement(mac, placement, static)
```